pyaedt/modules/Mesh.py

```diff
@@ -1101,35 +1101,44 @@ def assign_skin_depth(
         else:
             name = generate_unique_name("SkinDepth")
 
-        if maximum_elements is None:
-            restrictlength = False
-            maximum_elements = "1000"
+        if self._app.design_type == "Maxwell 2D":
+            seltype = "Edges"
+            props = OrderedDict(
+                {
+                    seltype: assignment,
```

Comment:
```suggestion
            props = OrderedDict(
                {
                    "Edges": assignment,
```
There is no real need to create `seltype` in this scope as it is never used later on. Once leaving it, only `name` and `props` are used. Keeping such variable could make someone think that an error was introduced in this refactoring as it gives the impression that `seltype` should be used after the `if` ... `else` ... statements.

Reply:
@SMoraisAnsys you are right! an oversight ;)